env.Dockerfile

```diff
@@ -0,0 +1,40 @@
+# *** Base *** #
+FROM python:3.10 AS base
```

Comment:
Were we always using 3.10 for docker, or 3.9?

Reply:
We had 3.9 to match the lowest python version supported by MrMustard but @ziofil uses 3.10 more often inside devcontainers

Reply:
If it were for me I'd use 3.12! Maybe now 3.11 is supported too

Reply:
I have updated the devcontainer definition so it will build with 3.10 _by default_, but you can specify another version if you please. To do this, just have the `MRMUSTARD_PYTHON_VERSION` environment variable set locally before building+opening the devcontainer (default will be 3.10).

Some notes:
- I feel like VS Code always refreshes something before building, so I'd put it in your `~/.bashrc`, ~/.zshrc` or whatever you use to be safe
- We can't build with 3.12 because tensorflow fails to install 🙃